WooCommerce/Classes/ViewRelated/Orders/Order Creation/NewOrderViewModel.swift

```diff
@@ -240,6 +241,15 @@ final class NewOrderViewModel: ObservableObject {
         })
     }
 
+    // MARK: Order notes data properties
+    @Published private(set) var customerNoteDataViewModel: CustomerNoteDataViewModel = .init(customerNotes: "")
+
+    lazy private(set) var noteViewModel = { OrderNotesViewModel(originalNote: customerNoteDataViewModel.customerNotes) }()
```

Comment:
We try to have variables before the initializer and methods after, especially private ones who go on a private extension.

I dunno why `customerDataViewModel ` is there :-/

Reply:
If this view model is our source of truth(holds the real data) I think it should be created with an empty string and not with the contents of `customerNoteDataViewModel` which I know also holds an empty string but makes the dependency relationship a bit confusing. 

What do you think?